frog/imports/ui/GraphEditor/Operators.js

```diff
@@ -20,8 +20,8 @@ export default connect(
           title={op.title}
           transparent={transparent}
           key={op.id}
-          x={coords[0]}
-          y={coords[1]}
+          x={coords[0] - 10}
+          y={coords[1] - 30}
```

Comment:
I really don't like the look of that fix. Seems that we will be offset again as soon as something new moves.